tests/test_plugins/discovery_test_plugin/tests/test_discovery.py

```diff
@@ -18,3 +18,13 @@ def test_number_of_imports(tmpdir: Path) -> None:
     with Path(os.environ["TMP_FILE"]) as f:
         txt = str(f.read_text())
         assert txt.split("\n").count("imported") == 1
+
+
+def test_skipped_imports(tmpdir: Path) -> None:
```

Comment:
did you actually see this being tripped without your fix?
this plugin does not contain any files starting with _.

Reply:
add a specific file and just validate it's not being imported directly.
no need to look at module import times. create some kind of side effect from the _module and test that.

Reply:
This test is running for all the plugins. You are right that nothing would break right now but once Ax fix lands, it will break.

We can do the side effect thing but since the change is specifically not importing the file during plugin discovery, it seemed quite natural.

Reply:
all installed plugins. which plugins are installed when it's running?

Reply:
Ahh got it. I had the plugins installed locally. Will take another stab.

Reply:
try to be as direct as possible when testing.